classes/services/queryBuilders/SubmissionListQueryBuilder.inc.php

```diff
@@ -44,6 +44,38 @@ public function filterBySections($sectionIds) {
 	 * @return object Query object
 	 */
 	public function appGet($q) {
+		$primaryLocale = \AppLocale::getPrimaryLocale();
+		$locale = \AppLocale::getLocale();
+
+		$this->columns[] = Capsule::raw('COALESCE(stl.setting_value, stpl.setting_value) AS section_title');
+		$this->columns[] = Capsule::raw('COALESCE(sal.setting_value, sapl.setting_value) AS section_abbrev');
+
+		$q->groupBy(Capsule::raw('COALESCE(stl.setting_value, stpl.setting_value)'));
+		$q->groupBy(Capsule::raw('COALESCE(sal.setting_value, sapl.setting_value)'));
+
+		$q->leftJoin('section_settings as stpl', function($join) use($primaryLocale) {
+			$join->on('s.section_id', '=', Capsule::raw("'stpl.section_id'"));
+			$join->on('stpl.setting_name', '=', Capsule::raw("'section_title'"));
+			$join->on('stpl.locale', '=', Capsule::raw("'{$primaryLocale}'"));
+		});
+
+		$q->leftJoin('section_settings as stl', function($join) use($locale) {
+			$join->on('s.section_id', '=', Capsule::raw("'stl.section_id'"));
+			$join->on('stl.setting_name', '=', Capsule::raw("'section_title'"));
+			$join->on('stl.locale', '=', Capsule::raw("'{$locale}'"));
```

Comment:
It does. Unfortunately, the bound params get compiled out of order otherwise, so we lose some of the benefits of the query builder here. But... (see below)